examples/Search_reranking_with_cross-encoders.ipynb

```diff
@@ -245,7 +245,7 @@
         "\n",
         "@retry(wait=wait_random_exponential(min=1, max=40), stop=stop_after_attempt(3))\n",
         "def document_relevance(query, document):\n",
-        "    response = openai.Completion.create(\n",
+        "    response = openai.chat.completions.create(\n",
```

Comment:
- import openai, upgrade to new version and change to cilent variable in the file.
- update the model to gpt-4, gpt-4 turbo, gpt-3.5-turbo

- there is no prompt parameter it's message

- response format.

- maybe check the whole file if any other change is needed.